chord_metadata_service/patients/api_views.py

```diff
@@ -170,32 +171,51 @@ def get_queryset(self):
 
 
 async def public_discovery_filter_queryset(
-    discovery: DiscoveryConfig, request: DrfRequest, dt_permissions: DataTypeDiscoveryPermissions, queryset: QuerySet
+    discovery: DiscoveryConfig,
+    request: DrfRequest,
+    dt_permissions: DataTypeDiscoveryPermissions,
+    queryset: QuerySet,
+    scope_repr: str,  # We pass disc
```

Comment:
this was to avoid having to deal with `EmptyConfig` type in two places, since scopes can return empty configs. but yeah, i agree this also isn't the best (and the comment is truncated/unfinished)